### PR TITLE
Documentation updates for 1.18

### PR DESCRIPTION
**Pull request by dzamo**
Numerous contributions to bring the Drill documentation up to speed with release 1.18.

### CODE REVIEW

**Comment by cgivre** [comment]
Could you also please add the following docs to the section about querying files?

- https://github.com/apache/drill/tree/master/contrib/format-excel
- https://github.com/apache/drill/tree/master/contrib/format-hdf5
- https://github.com/apache/drill/tree/master/contrib/format-spss
- https://github.com/apache/drill/tree/master/contrib/format-syslog
- https://github.com/apache/drill/tree/master/contrib/format-esri

Also, we have a few new storage plugins which we should add to the docs which are:
- https://github.com/apache/drill/tree/master/contrib/storage-http
- https://github.com/apache/drill/tree/master/contrib/storage-druid

**Comment by cgivre** [comment]
One more thing... take a look here https://github.com/apache/drill/tree/master/contrib/udfs for UDFs included with Drill.  Ideally these should be in the docs.

**Comment by cgivre** [comment]
@dzamo 
Any update?
Thanks

**Comment by dzamo** [comment]
Hi Charles, I've been travelling. I hope to return to this next week. 
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by cgivre** [comment]
Hi @dzamo 
I hope all is well.  Do you have any update on this?  Should we just go ahead and commit?

**Comment by dzamo** [comment]
If this isn't holding anything back and we can wait a few more days then let us, because I certainly have more to add.  My apologies for the stalling.  As COVID-19 lockdown lifted where I am, a period of intense travelling ensued.  But it concluded at the start of this week.

**Comment by cgivre** [comment]
@dzamo 
Thanks for getting back with me.  Would a week or so be a reasonable timetable for next review?

**Comment by dzamo** [comment]
@cgivre
Yes works for me.

File: _docs/configure-drill/configuration-options/030-planning-and-exececution-options.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ You can run the following query to see a list of options:
...
 
 	SELECT COUNT() AS num_of_sysopts FROM sys.options;
-	+-----------------+
+	|-----------------|
 	| num_of_sysopts  |
-	+-----------------+
+	|-----------------|
 	| 179             |
-	+-----------------+  
+	|-----------------|
```
[the remark this answers is not in the document]
Comment (dzamo) [reply _docs/configure-drill/configuration-options/030-planning-and-exececution-options.md:34]: I couldn't see any _major_ problem here.  I've corrected `###Drill-override...` to `### Drill-override...`

File: _docs/connect-a-data-source/plugins/114-image-metadata-format-plugin.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Image Metadata Format Plugin"
```
[the remark this answers is not in the document]
Comment (dzamo) [reply _docs/connect-a-data-source/plugins/114-image-metadata-format-plugin.md:2]: Absolutely.  I've moved the format plugins away from the storage plugins, and under Data Sources and File Formats which seems to me to be the best available place.  I think there are further other reorganisation opportunities like this, but they can be for future PRs.

**Comment by dzamo** [comment]
I think that may just be a wrap.

**Pull request merged**